src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/paragraph/helpers.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/prefer-default-export */
+import path from 'ramda/src/path';
+
+const boldWrap = text => `<bold>${text}</bold>`;
+
+const escapeDoubleQuotes = text => text.replace(/&quot;/g, '"');
+
+export const processBlock = _block => {
+  const block = _block;
+
+  if (path(['text'], block)) {
+    // escape quotes in all text
+    block.text = escapeDoubleQuotes(block.text);
+
+    // make introductions bold
+    if (block.type === 'paragraph' && block.role === 'introduction') {
```

Comment:
```suggestion
    if (block.role === 'introduction') {
```

The `block.type` will always be paragraph by the time we are in the paragraph helper